Exercises/STAM_Exercise.py

```diff
@@ -0,0 +1,59 @@
+__author__ = "Sol"
+
+
+import sys
+import os
+import shutil
+FILE = 1
+PATH = 2
+
+
+def main():
+    text_file = sys.argv[FILE]
+    path = sys.argv[PATH]
+    is_exist(text_file, path)
+    add_strings(text_file, path)
+
+
+def is_exist(text_file, path):
+    """
+    checks if the text file is a file
+    checks if the path exist
+    print the appropriate response
+    :param text_file: input text file
+    :param path: input path
+    :return: True/ False
+    """
+    if os.path.isfile(text_file):
+        if os.path.exists(path):
+            # if both exist and OK returns True
+            return True
+        print("The path not exist")
+        return False
+    print("This is not a text file")
+    return False
```

Comment:
read the code styling the main repo and fix your file accordingly.
(blank lines between logic blocks)